kernels/setup_cuda_env.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Do we need this file?

Reply:
I will remove this one, it's my local file, added accidentally